index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <script src="http://cdn.mathjax.org/mathjax/2.3-latest/MathJax.js?config=MML_HTMLorMML"></script>
```

Comment:
This same line would need to be in exercises. Also, should probably switch to 2.5. I pulled this from cnx production but I think they're moving to 2.5